packages/orbit-components/src/Box/tailwindClasses.ts

```diff
@@ -0,0 +1,224 @@
+import { QUERIES } from "../utils/mediaQuery/consts";
+
+enum WRAP {
+  WRAP = "wrap",
+  NO_WRAP = "nowrap",
+}
+
+export const wrapClasses: {
```

Comment:
this should be also part of the common folder, I guess

Reply:
I have considered that. But we actually have no other space where we use the `wrap` prop as string.

We only have a `wrap` prop in one more component: Stack. And it uses it as a boolean, so the logic there won't be accessing an object, but probably using a ternary or something. So I didn't find any value in having this on the common.

I am fine in moving it there, ofc, as I also think it is something to be reusable, but just saying that, for now, it won't be used anywhere else.